composer.json

```diff
@@ -12,9 +12,22 @@
 	"prefer-stable": true,
 	"require": {
 		"dnadesign/silverstripe-elemental": "dev-ss4",
-		"silverstripe/userforms": "dev-master"
+		"silverstripe/userforms": "5.0.x-dev",
+        "silverstripe/vendor-plugin": "^1.0"
 	},
 	"extra": {
-		"installer-name": "elemental-userforms"
-	}
+		"installer-name": "elemental-userforms",
+        "branch-alias": {
+            "dev-master": "1.x-dev"
+        },
+        "expose": [
+            "images"
+        ]
+    },
+    "autoload": {
+        "psr-4": {
+            "DNADesign\\Elemental\\UserForms\\": "src/",
+            "DNADesign\\Elemental\\UserForms\\Tests\\": "tests/php/"
```

Comment:
I think the namespace should be `DNADesign\ElementalUserforms` so it matches the vendor\package structure